README.md

```diff
@@ -216,6 +216,18 @@ resource = dataset.create_remote(
 # to update the file of a static resource
 resource.update({"title": "New title"}, file_to_upload="path/to/your/new_file.txt")
 ```
+
+You can also create a static resource from a file using its URL by setting `from_url=True`, in which case the `file_name` and `mime` arguments are required:
```

Comment:
I would add a warning that in most cases a remote URL is a better idea if the target URL is stable